sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -271,6 +276,11 @@ def __init__(self, client=None):
     # randomized prefix for row IDs.
     self._row_id_prefix = '' if client else uuid.uuid4()
     self._temporary_table_suffix = uuid.uuid4().hex
+    self._latency_histogram_metric = Metrics.histogram(
```

Comment:
We should make this clear that this isn't fully supported as a user metric type in the SDK, or in any Runners. We may need to relocate this class from Metrics.histogram to somewhere else i.e. InternalMetrics.histogram

@pabloem any ideas what we ought to do here? This metric isn't fully implement but the API allows us to log the metric locally.

Reply:
I agree with Alex. We have no plans to support this in any way by runners. It may be much simpler to just create a basic  Histogram metric object that does not integrate with the whole user metrics framework.
I feel that integrating with the user metrics framework will generate confusion for users and developers looking at it in the future, as it is not intended to be a user metric.

@ihji can you talk about your motivation to integrate with the user metrics framework?

Reply:
We do plan on building a few system metrics for GCP IO API request call latency which use histograms. Though, it doesn't need to use the user metric framework itself, its a reasonable way to implement it to use a lot of the same primitives.

I asked @ihji to use the metric framework for the logger, as I will be adding a latency histogram metrics for these BigQuery API request calls.

Reply:
Pablo and I discussed this. A bit more. @ihji Would you mind please moving the Metrics.histogram() into a "internal" folder. We don't want to suggest that users use this yet, if we are only using it for system/internal style metrics/monitoring.

Reply:
done.